Data/market-hours/market-hours-database.json

```diff
@@ -23329,9 +23329,47 @@
         }
       ],
       "saturday": [],
-      "holidays": [],
+      "holidays": [
+	"1/1/2007",
+        "1/1/2008",
+        "1/1/2009",
+        "1/1/2010",
+        "1/1/2012",
+        "1/1/2013",
+        "1/1/2014",
+        "1/1/2015",
+        "1/1/2016",
+        "1/1/2017",
+        "1/1/2018",
+        "1/1/2019",
+        "1/1/2020",
+        "1/1/2021",
+        "1/1/2024",
+        "1/1/2025",
+        "12/25/2007",
+        "12/25/2008",
+        "12/25/2009",
+        "12/25/2011",
+        "12/25/2012",
+        "12/25/2013",
+        "12/25/2014",
+        "12/25/2015",
+        "12/25/2016",
+        "12/25/2017",
+        "12/25/2018",
+        "12/25/2019",
+        "12/25/2020",
+        "12/25/2023",
+        "12/25/2024",
+        "12/25/2025"
+      ],
       "earlyCloses": {
-        "12/31/2018": "17:00:00"
+        "12/31/2018": "21:00:00",
```

Comment:
Why was the time changed from "17:00:00" to "21:00:00"?
Early close time comparisons are done using the exchange time zone:
https://github.com/QuantConnect/Lean/blob/master/Common/Securities/SecurityExchangeHours.cs#L336

Reply:
Looks like early closes are also missing for `"12/24/20xx": "17:00:00"`